tests/Aspire.Hosting.Azure.Tests/AzurePostgresExtensionsTests.cs

```diff
@@ -166,28 +166,31 @@ public async Task AddAzurePostgresWithPasswordAuth(bool specifyUserName, bool sp
         var expectedManifest = $$"""
             {
               "type": "azure.bicep.v0",
-              "connectionString": "{postgres-data.secretOutputs.connectionString}",
+              "connectionString": "{postgres-data-kv.secrets.postgres-data--connectionString}",
```

Comment:
Is there a change queued up on the `azd` side to process `.secrets` instead of secret outputs?

Reply:
cc @vhvb1989